configuration/configuration_reference.md

```diff
@@ -877,6 +857,211 @@ can be used to **customize how config repo plugin works when parsing this specif
 
 [top](#top)
 
+## &lt;svn&gt; {#config-repo-svn}
+
+The config repo `<svn>` material element specifies the location of your code base in Subversion repository.
+
+### Attributes
+
+| Attribute | Required | Description |
+|-----------|----------|-------------|
+| url | Yes | URL for the remote repository, for example: 'http://www.thoughtworks-studios.com/go-agile-release-management/svn/myproject/trunk'. Go supports the following protocols for subversion: http, https, svn and svn+ssh, but does not support 'file:///'. |
+| username | No | The user account for the remote repository. |
+| password | No | The password for the specified user |
+| encryptedPassword | No | The encrypted password for the specified user |
+| checkexternals | No | The default value is false, the value should be either one of true/false or 1/0. 'true' or '1' means that the changes of externals will trigger the pipeline automatically. |
+| materialName | Required if this material is referenced in pipeline labeltemplate | The name to identify a material. Material name can contain the following characters: a-z, A-Z, 0-9, fullstop, underscore and hyphen, but whitespace is not allowed. A material name is case insensitive and starting with fullstop is invalid. It needs to be unique within a pipeline. The max length is 255 characters. |
+| autoUpdate | No | By default Go polls the repository for changes automatically. If autoUpdate is set to false then Go will not poll the repository for changes. Instead it will check for changes only when you trigger a pipeline that contains this material or it receives a notification through a post-commit hook. If the same material is specified more than once in the configuration file, all of them must have the same value for autoUpdate. |
+
+### Notes:
+
+Go cannot automatically accept svn SSL certificates. If you are using https for svn repository, you have to go to the Server and each Agent,
+and as the user 'go' do a command "svn update" to store the certificates in the cache permanently.
+
+### Examples:
+
+For a Go Agent on linux with the following configuration:
```

Comment:
This again seems like a copy from the existing svn material, not sure what this means, esp since there is no agent component for a config repo